tetris.py

```diff
@@ -178,7 +186,22 @@ def block_mover(self):
             self.current_block_center = (x, y + 1)
 
     def block_rotator(self, event):
-        print("TODO: should rotate")
+        self.index += 1
+        self.index %= 4
+        if self.current_block_shape == "I":
+            return self.get_current_block[self.index]
+        if self.current_block_shape == "L":
+            return self.get_current_block[self.index]
+        if self.current_block_shape == "L_rev":
+            return self.get_current_block[self.index]
+        if self.current_block_shape == "O":
+            return
+        if self.current_block_shape == "E":
+            return self.get_current_block[self.index]
+        if self.current_block_shape == "Z":
+            return self.get_current_block[self.index]
+        if self.current_block_shape == "Z_rev":
+            return self.get_current_block[self.index]
```

Comment:
I suggest deleting all this, since incrementing `index` is enough to make `get_current_block()` to do the right thing, and hence get the block to show up as rotated:

```suggestion
```